src/common-components/search-bar/reducers/search-results-reducer.js

```diff
@@ -0,0 +1,20 @@
+import { FETCH_SEARCH_RESULTS, DELETE_SEARCH_RESULTS, NO_SEARCH_RESULTS } from '../actions';
+
+export default function(state = { next: null, course_type: 'none', results: [] }, action) {
+  switch (action.type) {
+    case FETCH_SEARCH_RESULTS:
+      let { next, results } = action.payload.data;
+      let { course_type } = action.payload;
+      state = { next, course_type, results: [...state.results, ...results] }
+      return state;
+      break;
+
+    case DELETE_SEARCH_RESULTS:
+      return { next: null, course_type: 'none', results: [] };
+      break;
+    case NO_SEARCH_RESULTS:
```

Comment:
Does the api not search in resource titles? Searching for 'django' returns no results when there are a lot of django resources in the results for 'python' search.

Reply:
Again, not a problem on frontend, ask paritosh about it.